bgpd/bgp_bmp.c

```diff
@@ -1679,6 +1680,22 @@ static int bmp_process(struct bgp *bgp, afi_t afi, safi_t safi,
 	return 0;
 }
 
+static int bmp_nht_path_valid(struct bgp *bgp, struct bgp_path_info *path, bool valid)
+{
+	struct bgp_dest *dest = path->net;
+	struct bgp_table *table;
+
+	if (frrtrace_enabled(frr_bgp, bmp_nht_path_valid)) {
+		char pfxprint[PREFIX2STR_BUFFER];
+
+		prefix2str(&dest->rn->p, pfxprint, sizeof(pfxprint));
+		frrtrace(6, frr_bgp, bgp_nht_path_valid, bgp, path, pfxprint, valid);
```

Comment:
Where is this `bgp_nht_path_valid` tracepoint defined? And it should take 3 arguments, not 6?

Reply:
you are right, I compiled this time with lttng, and did the necessary fixes.